python/foglamp/services/core/api/plugins/remove.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+import platform
+import os
+from aiohttp import web
+import logging
+from foglamp.common import logger
+from foglamp.common.plugin_discovery import PluginDiscovery
+from foglamp.services.core.api.plugins import common
+
+__author__ = "Rajesh Kumar"
+__copyright__ = "Copyright (c) 2019, Dianomic Systems Inc."
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+_logger = logger.setup(__name__, level=logging.INFO)
+
+valid_plugin = ['north', 'south', 'filter', 'notificationDelivery', 'notificationRule']
+
+
+async def plugin_delete(request):
+    """
+    Remove plugin from foglamp
+
+    '''
+    EndPoint: curl -X DELETE http://host-ip:port/foglamp/plugins/{type}/{name}/remove
+    '''
+    Example:
+        curl -X DELETE http://host-ip:port/foglamp/plugins/south/sinusoid/remove
```

Comment:
There is no need to have the /remove on the end of the URL, this is a DELETE operation